test/milton/MiltonShouldClosePosition.t.sol

```diff
@@ -0,0 +1,4797 @@
+            address(_daiMockedToken),
+            address(_ipTokenDai),
+            address(mockCase3MiltonDai),
+            address(miltonStorageDai),
+            address(stanleyDai)
+        );
+        prepareApproveForUsersDai(_users, _daiMockedToken, address(mockCase0JosephDai), address(mockCase3MiltonDai));
+        prepareMilton(mockCase3MiltonDai, address(mockCase0JosephDai), address(stanleyDai));
+        prepareJoseph(mockCase0JosephDai);
+        prepareIpToken(_ipTokenDai, address(mockCase0JosephDai));
+        vm.prank(_userOne);
+        iporOracle.itfUpdateIndex(address(_daiMockedToken), TestConstants.PERCENTAGE_3_18DEC, block.timestamp);
+        vm.prank(_liquidityProvider);
+        mockCase0JosephDai.itfProvideLiquidity(4 * TestConstants.USD_28_000_18DEC, block.timestamp);
+        uint256[] memory payFixedSwapIds = new uint256[](2);
+        payFixedSwapIds[0] = 1;
+        payFixedSwapIds[1] = 3;
+        uint256[] memory receiveFixedSwapIds = new uint256[](2);
+        receiveFixedSwapIds[0] = 2;
+        receiveFixedSwapIds[1] = 4;
+        openSwapPayFixed(
+            _userTwo,
+            block.timestamp,
+            TestConstants.TC_TOTAL_AMOUNT_10_000_18DEC,
+            TestConstants.PERCENTAGE_6_18DEC,
+            TestConstants.LEVERAGE_18DEC,
+            mockCase3MiltonDai
+        );
+        openSwapReceiveFixed(
+            _userTwo,
+            block.timestamp,
+            TestConstants.TC_TOTAL_AMOUNT_10_000_18DEC,
+            TestConstants.PERCENTAGE_1_18DEC,
+            TestConstants.LEVERAGE_18DEC,
+            mockCase3MiltonDai
+        );
+        openSwapPayFixed(
+            _userThree,
+            block.timestamp,
+            TestConstants.TC_TOTAL_AMOUNT_10_000_18DEC,
+            TestConstants.PERCENTAGE_6_18DEC,
+            TestConstants.LEVERAGE_18DEC,
+            mockCase3MiltonDai
+        );
+        openSwapReceiveFixed(
+            _userThree,
+            block.timestamp,
+            TestConstants.TC_TOTAL_AMOUNT_10_000_18DEC,
+            TestConstants.PERCENTAGE_1_18DEC,
+            TestConstants.LEVERAGE_18DEC,
+            mockCase3MiltonDai
+        );
+        // when
+        vm.prank(_userThree);
+        mockCase3MiltonDai.itfCloseSwaps(
+            payFixedSwapIds, receiveFixedSwapIds, block.timestamp + TestConstants.PERIOD_28_DAYS_IN_SECONDS
+        );
+        // then
+        IporTypes.IporSwapMemory memory payFixedSwapOne = miltonStorageDai.getSwapPayFixed(1);
+        IporTypes.IporSwapMemory memory receiveFixedSwapTwo = miltonStorageDai.getSwapReceiveFixed(2);
+        IporTypes.IporSwapMemory memory payFixedSwapThree = miltonStorageDai.getSwapPayFixed(3);
+        IporTypes.IporSwapMemory memory receiveFixedSwapFour = miltonStorageDai.getSwapReceiveFixed(4);
+        assertEq(payFixedSwapOne.state, TestConstants.ZERO);
+        assertEq(receiveFixedSwapTwo.state, TestConstants.ZERO);
+    
```

Comment:
Missing test "should close position, DAI, when amount exceeds balance milton on DAI token" fix it